### PR TITLE
feat: bootstrap phase

### PR DESCRIPTION
**Pull request by gtsorbo**

### CODE REVIEW

File: 1-bootstrap/variables.tf [hunk 1]
```diff
@@ -15,6 +15,5 @@
  */
 
 variable "project_id" {
-  description = "The ID of the project in which to provision resources."
-  type        = string
+  description = "Project ID for intiial resources"
```
Comment (apeabody) [inline 1-bootstrap/variables.tf:18]:
```suggestion
  description = "Project ID for initial resources"
```

File: 1-bootstrap/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  - Build Artifacts bucket, to store any artifacts generated during the build process, such as `.tfplan` files
+  - Build Logs bucket, to store the logs from the build process
+- 1 service account for executing the Cloud Build build process
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| project\_id | Project ID for intiial resources | `any` | n/a | yes |
```
Comment (apeabody) [inline 1-bootstrap/README.md:23]:
```suggestion
| project\_id | Project ID for initial resources | `any` | n/a | yes |
```

**Review by apeabody — APPROVED** [review]
nit: typo

File: 1-bootstrap/main.tf [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  version  = "~> 7.0"
+
+  project_id               = var.project_id
+  tf_repo_uri              = google_sourcerepo_repository.gcp_repo[each.key].url
+  tf_repo_type             = "CLOUD_SOURCE_REPOSITORIES"
+  artifacts_bucket_name    = "${each.value.bucket_prefix}-build-${var.project_id}"
+  create_state_bucket_name = "${each.value.bucket_prefix}-state-${var.project_id}"
+  log_bucket_name          = "${each.value.bucket_prefix}-logs-${var.project_id}"
+
+  #   cloudbuild_plan_filename  = "cloudbuild-tf-plan.yaml"
```
Comment (apeabody) [inline 1-bootstrap/main.tf:60]: Hmm, likely adding one of these will workaround the plan drift by populating `options`:

https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/v7.0.0/modules/tf_cloudbuild_workspace/cb.tf#L109C20-L109C44

We could also look into changing the upstream module, could be a recent provider change.

Comment (gtsorbo) [reply 1-bootstrap/main.tf]: Good call, will do that

**Pull request merged**